src/server/server.ts

```diff
@@ -83,10 +85,13 @@ interface ServeOptions {
   port: number
 }
 
+// eslint-disable-next-line complexity, max-statements
 const serve = async ({
   certificatePath,
   debug,
   distImportMapPath,
+  edgeInspect,
```

Comment:
What happens if we receive both `edgeInspect` and `edgeInspectBrk`?

Rather than receiving two flags that can be either strings or booleans, what do you think of receiving a single variable of an `InspectSettings` type looking something like:

```ts
interface InspectSettings {
  // Whether inspect mode is enabled
  enabled: boolean

  // Whether to pause on breakpoints (i.e. "--brk")
  pause: boolean

  // Host/port override (optional)
  address?: string
}
```

CLI would then compile this object based on the flags it receives and pass it along to Edge Bundler.

Reply:
It defaults to --inspect-brk if both are received so I've added a conflict check to CLI so both are not passed at the same time. Switched edge-bundler over to a single object as well.